vtex-sites/storecomponents.store
# feat: use Thumbor on product summary image

# Description
As title says ...

# How to test?
Open any search page and make sure the product summary images are showing up. Also, make sure that the search suggestions as well an the cart images are ok. Basically, check every place the product summary is used
